coda_docs.gemspec

```diff
@@ -40,6 +40,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'pry-byebug'
   spec.add_development_dependency 'rake', '~> 13.0'
   spec.add_development_dependency 'rspec', '~> 3.0'
-  spec.add_dependency 'activesupport', '~> 7.0.8'
+  spec.add_dependency 'activesupport'
```

Comment:
Could we pessimisstically lock to 6? According to Bundler docs, a pessimistic lock is best practice. 
![image](https://github.com/mavenlink-solutions/coda_docs/assets/1417659/47f9981c-1429-4227-a4b0-e5ef06fedcbf)
https://guides.rubygems.org/patterns/#pessimistic-version-constraint